### PR TITLE
Docs: What's new in 7.2 

### PR DESCRIPTION
**Pull request by torkelo**
* [x] Custom date formats and screenshot
* [x] Table column filters and screenshot
* [ ] More Field Overrides & UI update (needs text update, maybescreenshot)
* [ ] New and improved transforms (needs screenshots and text) 
* [ ] XXX

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -3,7 +3,6 @@
 IMAGE = grafana/docs-base:latest
 
 docs:
-	docker pull ${IMAGE}
```
Comment (torkelo) [inline docs/Makefile:6]: no idea how others test with new images, but I have to always remove this so that it does not overwrite my locally built image

**Comment by torkelo** [comment]
Current state 
localhost_3002_docs_grafana_latest_guides_whats-new-in-v7-2_ (1)

**Comment by mckn** [comment]
transformations

@torkelo would this work for the transformations?

**Comment by torkelo** [comment]
yes

**Pull request merged**